hax/treemakers/posrec.py

```diff
@@ -55,6 +54,8 @@ class PositionReconstruction(TreeMaker):
 
     def __init__(self):
 
+        #from keras.models import model_from_json
```

Comment:
Why commented?

Reply:
@jpienaar13 Can remove if you moved it below.